api/.eslintrc

```diff
@@ -6,6 +6,9 @@
   "rules": {
     "no-console": "error"
   },
+  "parserOptions": {
+    "ecmaVersion": 2017,
```

Comment:
We still support node 8 which [doesn't have full ES2017 support](https://node.green/#ES2017) so I think this is dangerous. What feature do you need/want?

Reply:
That link you sent only shows up to node 8.9.4.  But we support [>= node 8.11](https://github.com/medic/medic/blob/master/api/package.json#L11) and have 8.11.1 running in the docker container.

Node 8.10 supports full ES2017.  You can see [SharedBufferArrays are supported in 8.10](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer) as are [Atomics](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics).  I just want to use async/await. I've tested this code in the docker container.

Reply:
Ok, looks fine. Let's go for it. If it bites us we can roll it back... Can't wait for 4.0 to bump dependencies!